template/analytics.notify.cve/body.html

```diff
@@ -101,18 +101,23 @@
           <th style="text-align: center;">Current Version</th>
           <th>CVE ID(s) and corresponding CVSS Score(s)</th>
         </tr>
+        {{ range $depsInd, $dependency := .custom.vulnerable_deps }}
         <tr>
-          <td>#01</td>
-          <td>com.logback.core</td>
-          <td style="text-align: center;">2.0</td>
-          <td>CVE-2016-7643<span style="font-weight: 600;padding-left:2px;">[9/10]</span>, CVE-2016-7643<span style="font-weight: 600;padding-left:2px;">[9/10]</span></td>
-        </tr>
-        <tr>
-          <td>#02</td>
-          <td>com.logback.web</td>
-          <td style="text-align: center;">1.5</td>
-          <td>CVE-2016-7643<span style="font-weight: 600;padding-left:2px;">[9/10]</span>, CVE-2016-7643<span style="font-weight: 600;padding-left:2px;">[9/10]</span></td>
+          <td>#{{inc $depsInd}}</td>
+          <td>{{$dependency.name}}</td>
+          <td>{{$dependency.version}}</td>
+          <td>
+            {{ $cvesArr := $dependency.cves }}
+            {{ range $ind, $cveMap := $cvesArr }}
+              {{ if (eq $ind 0) }}
+              {{ $cveMap.CVE }} <b>[{{ $cveMap.CVSS }}/10]</b>
+              {{ else }}
+              ,&nbsp;{{ $cveMap.CVE }} <b>[{{ $cveMap.CVSS }}/10]</b>
```

Comment:
There is some 'bug' here. The data comes out with a " " in front of ",":

    CVE-2016-7643 [8.0/10] , CVE-2016-7643 [5.0/10]

Should be;

    CVE-2016-7643 [8.0/10], CVE-2016-7643 [5.0/10]